ports/glslang/portfile.cmake

```
@@ -4,12 +4,16 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO KhronosGroup/glslang
     REF "${VERSION}"
-    SHA512 678df76a6f23b9da93f111fc7e6db57b7f6bf34661b077f9259a0a77d6c023b4d2e3c1cd60b3f9fc15fe69f25cdcb19877e88a50771d3d5275e32574eaefc056
+    SHA512 e16b01925a657750733a2973dc803fc3910a3a169ae276af205de6cb1bf0536fd2dbb63c5fd4fc10f800ba95f71bce673417121ad640cb9c964f291596c80025
     HEAD_REF master
+    PATCHES
+        cmake.patch # https://github.com/KhronosGroup/glslang/pull/3406 and https://github.com/KhronosGroup/glslang/pull/3420
```

```suggestion
        cmake.patch # Remove this on next version 
```
It looks like the patch has been merged. :)